### PR TITLE
Plumbing related equipment's added

### PR DESCRIPTION
**Pull request by darshanjain12**
Added Tanks ,Organic waste compactor system ,Pumps for plumbing category

### CODE REVIEW

File: ontology/yaml/resources/PLUMBING/entity_types/ABSTRACT.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  - failed_alarm
+  - lost_power_alarm
+  - control_mode
+
+
+LVLM:
+  description: "Level Monitoring for tanks."
+  is_abstract: true
+  uses:
+  - level_percentage_status
```
[the remark this answers is not in the document]
Comment (tasodorff) [reply ontology/yaml/resources/PLUMBING/entity_types/ABSTRACT.yaml:32]: and status is used for multistates, not continuous readings. sensor is used in that case

**Review by tasodorff — CHANGES_REQUESTED** [review]
failing validation. see comments.

**Comment by mschulze17** [comment]
@tasodorff back to you

File: ontology/yaml/resources/PLUMBING/entity_types/GENERALTYPES.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+TK:
+  description: "Tank water system used to store water"
+  is_abstract: true
+  implements:
+  - EQUIPMENT
+
+OWCP:
```
Comment (tasodorff) [inline ontology/yaml/resources/PLUMBING/entity_types/GENERALTYPES.yaml:21]: this isnt BDNS compliant; please confirm this is the acronym to use for the device.

File: ontology/yaml/resources/PLUMBING/entity_types/PMP.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+# limitations under the License.
+# canonical types
+
+########################
+### Canonical Types  ###
+########################
+
+
+PMP_SS:
+  description: "Basic water pump ."
```
Comment (tasodorff) [inline ontology/yaml/resources/PLUMBING/entity_types/PMP.yaml:22]: nit: remove trailing space after final word to full stop.

**Review by tasodorff — CHANGES_REQUESTED** [review]
see comments

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 4]
```diff
@@ -1141,6 +1141,12 @@ literals:
 - potable_hot_leaving_water_temperature_sensor
 - potable_hot_leaving_water_temperature_setpoint
 
+## New fields for tanks
+- level_status:
+ - LOW
```
Comment (tasodorff) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:1146]: fix this as well, its causing a parsing error. match the indents used for states above.

File: ontology/yaml/resources/PLUMBING/entity_types/GENERALTYPES.yaml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+WSTC:
+  description: " Waste Compactor system used for compacting organic waste essential for conversion into manure."
+  is_abstract: true
+  implements:
+  - EQUIPMENT
+
+PMP:
+  description: "Tag for pumps. Pumps move liquid (water, glycol, liquid CO2, etc.) from one part of a system
+    to another.Used for (irrigation,domestic purposes) different usecase then HVAC."
```
Comment (tasodorff) [inline ontology/yaml/resources/PLUMBING/entity_types/GENERALTYPES.yaml:29]: nit: space

Comment (tasodorff) [inline ontology/yaml/resources/PLUMBING/entity_types/GENERALTYPES.yaml:29]: nit: "use case than"

File: ontology/yaml/resources/PLUMBING/entity_types/WSTC.yaml [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+# limitations under the License.
+# canonical types
+
+########################
+### Canonical Types  ###
+########################
+
+
+WSTC_SS:
+  description: "Basic waste compactor system ."
```
Comment (tasodorff) [inline ontology/yaml/resources/PLUMBING/entity_types/WSTC.yaml:22]: nit: remove space

**Review by tasodorff — DISMISSED** [review]
bunch of nits on typos and formatting, but no substantial changes requested.

File: ontology/yaml/resources/PLUMBING/entity_types/ABSTRACT.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (darshanjain12) [reply ontology/yaml/resources/PLUMBING/entity_types/ABSTRACT.yaml]: Done required changes

File: ontology/yaml/resources/PLUMBING/entity_types/GENERALTYPES.yaml (hunk 2 above)
Comment (darshanjain12) [reply ontology/yaml/resources/PLUMBING/entity_types/GENERALTYPES.yaml]: Done required changes

File: ontology/yaml/resources/fields/telemetry_fields.yaml (hunk 4 above)
Comment (darshanjain12) [reply ontology/yaml/resources/fields/telemetry_fields.yaml]: Done

File: ontology/yaml/resources/PLUMBING/entity_types/WSTC.yaml (hunk 6 above)
Comment (darshanjain12) [reply ontology/yaml/resources/PLUMBING/entity_types/WSTC.yaml:22]: Thanks you for correcting .

File: ontology/yaml/resources/PLUMBING/entity_types/GENERALTYPES.yaml (hunk 5 above)
Comment (darshanjain12) [reply ontology/yaml/resources/PLUMBING/entity_types/GENERALTYPES.yaml:29]: Thanks you for correcting .Done the changes

**Comment by darshanjain12** [comment]
> bunch of nits on typos and formatting, but no substantial changes requested.
Thank you done with the changes.

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**